scripts/schemas.py

```diff
@@ -0,0 +1,63 @@
+from typing import Literal
+from pydantic import BaseModel, HttpUrl, ConfigDict, Field
+
+
+class Scopes(BaseModel):
+    """
+    Defines the responsibilities of a Conda sub-team.
+    This structure is nested within the main CondaSubTeam model.
+    """
+
+    model_config = ConfigDict(extra="forbid")
+
+    codeowners: list[str] | None = ...
+    """The GitHub projects this team owns"""
+
+    other: list[HttpUrl] | None = ...
+    """Other responsibilities of this team"""
+
+
+class CondaSubTeam(BaseModel):
+    """
+    Model for defining Conda Sub-Teams based on the governance structure.
+    """
+
+    model_config = ConfigDict(title="Conda Sub-Teams", extra="forbid")
+
+    name: str = ...
+    """The team name in GitHub"""
+
+    description: str = ...
+    """The team description in GitHub"""
+
+    charter: Literal["dynamic", "static", "project"] = ...
+    """
+    The team charter type, see [dynamic](https://github.com/conda/governance#dynamic-charter),
+    [static](https://github.com/conda/governance#static-charter),
+    and [project](https://github.com/conda/governance#project-teams-details) requirements.
+    """
+
+    requirements: str | None = ...
+    """Special requirements for team membership"""
+
+    scopes: Scopes = ...
+    """Team responsibilities"""
+
+    links: list[HttpUrl] = ...
+    """Important links, e.g. the issue/PR proposing the team creation"""
+
+    members: dict[str, HttpUrl | None] = Field(..., min_length=1)
```

Comment:
should we indicate who is a steering council member?

```yaml
name: ...
steering:
  GITHUB_ID: URL | None
  ...
members:
  GITHUB_ID: URL | None
  ...
emeritus:
  GITHUB_ID: URL | None
  ...
```

or should that be a separate assertion that each subteam has at least one steering council members